### PR TITLE
Fix small issues after rewrite

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: internal/handlers/jsonb1/msg_count.go [hunk 1]
```diff
@@ -33,29 +32,15 @@ func (s *storage) MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
 
 	unimplementedFields := []string{
 		"skip",
-		"returnKey",
```
Comment (seeforschauer) [inline internal/handlers/jsonb1/msg_count.go:36]: started to support this or it was added by a mistake?

File: internal/handlers/jsonb1/msg_count.go [hunk 2]
```diff
@@ -69,14 +54,11 @@ func (s *storage) MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
...
-	var filter, sort *types.Document
+	var filter *types.Document
 	var limit int32
 	if filter, err = common.GetOptionalParam(document, "query", filter); err != nil {
 		return nil, err
 	}
-	if sort, err = common.GetOptionalParam(document, "sort", sort); err != nil {
-		return nil, err
-	}
 	if limit, err = common.GetOptionalParam(document, "limit", limit); err != nil {
```
Comment (seeforschauer) [inline internal/handlers/jsonb1/msg_count.go:62]: limit is not necessary for count, isn't it?

File: internal/handlers/jsonb1/msg_count.go (hunk 1 above)
Comment (AlekSi) [reply internal/handlers/jsonb1/msg_count.go:36]: It was a part of "find or count" that was copied to count by mistake

**Review by seeforschauer — CHANGES_REQUESTED** [review]
pls remove limit from count

**Review by seeforschauer — APPROVED** [review]

**Pull request merged**